docs/reference-guides/interactivity-api/core-concepts/undestanding-global-state-local-context-and-derived-state.md

```diff
@@ -837,6 +837,68 @@ Whenever you have interactive blocks that rely on global state or local context
 -   **Read-Only References:** Both `getServerState()` and `getServerContext()` return read-only objects. You can use those objects to update the global state or local context.
 -   **Callback Integration:** Incorporate these functions within your store [callbacks](/docs/reference-guides/interactivity-api/api-reference.md#accessing-data-in-callbacks) to react to state and context changes. Both `getServerState()` and `getServerContext()` return reactive objects. This means that their watch callbacks will only trigger when the value of a property changes. If the value remains the same, the callback will not re-trigger.
 
+## Config
+
+**Config** in the Interactivity API refers to static configuration data that is serialized from the server to the client. Unlike global state or local context, config values are **not reactive** - they don't trigger UI updates when changed, and they're intended to remain constant throughout the client-side lifecycle.
```

Comment:
```suggestion
**Config** in the Interactivity API refers to static configuration data that is serialized from the server to the client. Unlike global state or local context, config values are **not reactive** - they don't trigger UI updates and remain constant throughout the client-side lifecycle.
```
If they can't be changed, let's not mention changing them :)

Reply:
Changed, thanks!